www/_posts/2024-11-01-cordova-lib-release-12.0.2.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova Lib 12.0.2 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are excited to announce the release of `cordova-lib`, one of our key CLI sub-dependencies! This package enables a range of operations, including adding and removing platforms and plugins.
```

Comment:
I know the idea is to give a quick overview of what the package is for, but "enables" feels like an awkward word here. Maybe "handles" is better?

Could also say something like "This releases has bug fixes for a range of operations, including ..."

Reply:
I went ahead and reverted it to the original default template where it just says we released but not giving a brief description of the package.